src/vm/hints/hint_utils.rs

```diff
@@ -28,6 +28,19 @@ pub fn get_int_from_scope(vm: &mut VirtualMachine, name: &str) -> Option<BigInt>
     val
 }
 
+pub fn get_int_from_scope_ref<'a>(
+    vm: &'a mut VirtualMachine,
```

Comment:
Does this need to be mutable?

Reply:
resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/333/commits/5a9bd80e9b1ee4158042e209384e3372f6e48792